fastapi/utils.py

```diff
@@ -38,11 +38,10 @@ def get_model_definitions(
     *, flat_models: Set[Type[BaseModel]], model_name_map: Dict[Type[BaseModel], str]
 ) -> Dict[str, Any]:
     definitions: Dict[str, Dict] = {}
-    for model in flat_models:
-        m_schema, m_definitions = model_process_schema(
+    for model in sorted(flat_models, key=lambda x: x.__name__):
```

Comment:
I added the sorting (by name) here to ensure reproducibility of the test I added. I assume the cost of this sort is negligible for practical purposes (given this is just used to generate the schema).